src/app/learnocaml_index_main.ml

```diff
@@ -637,23 +637,14 @@ let set_string_translations () =
        Manip.setInnerHtml (find_component id) text)
     translations
 
-let set_lang () =
-  match Js.Optdef.to_option (Dom_html.window##.navigator##.language) with
-  | Some l -> Ocplib_i18n.set_lang (Js.to_string l)
-  | None ->
-      match Js.Optdef.to_option (Dom_html.window##.navigator##.userLanguage)
-      with
-      | Some l -> Ocplib_i18n.set_lang (Js.to_string l)
-      | None -> ()
-
 let () =
```

Comment:
The similarity between `learnocaml_index_main.ml` and `learnocaml_exercise_main.ml` is striking. Can these two files share more code?